synapse/config/ratelimiting.py

```diff
@@ -143,6 +156,14 @@ def generate_config_section(self, **kwargs):
         #rc_admin_redaction:
         #  per_second: 1
         #  burst_count: 50
+        #
+        #rc_joins:
+        #  local:
+        #    per_second: 0.1
+        #    burst_count: 3
+        #  remote:
+        #    per_second: 0.01
+        #    burst_count: 3
```

Comment:
Wouldn't we want to allow faster local joins than remote joins?

Reply:
And we do! The burst count is the same but the rate is larger for locals

Reply:
Ah, I read these both as seconds per request instead of requests per second. Ignore me. 😢